tx/textual/internal/testdata/encode.json

```diff
@@ -0,0 +1,34 @@
+[
+    {
+        "screens": [],
+        "encoding": "80"
+    },
+    {
+        "screens": [{}],
+        "encoding": "81a0"
+    },
```

Comment:
```suggestion
    },
    {
        "screens": [{"indent": 0},{"expert": false}],
        "encoding": "81a0"
    },
```

Reply:
Done. Added the case for an explicitly empty text value, and the encoding will be an array of empty objects.